rx-java/src/main/asciidoc/vertx-rx/java/sql.adoc

```diff
@@ -0,0 +1,36 @@
+== SQL Transaction management with RxJava
+
+Managing SQL transactions manually with the Rxified `SQLConnection` requires quite a lot of boilerplate code.
+
+Vert.x provides __observable transformers__ that you can apply to your flows with http://reactivex.io/RxJava/1.x/javadoc/rx/Observable.html#compose-rx.Observable.Transformer-[`compose`] in order to make them transactional:
+
+* `InTransactionObservable`
+* `InTransactionSingle`
+* `InTransactionCompletable`
+
+These _transformers_ wrap the corresponding source of events with SQL transaction management. Let's take an example.
+
+In your music library application, you need to insert a row in table `albums`, then some rows in table `tracks`.
+These two steps shall be part of the same atomic transaction.
```

Comment:
transactions are always atomic :-)

Reply:
I guess the point was to insist on the A of ACID here